core/arch/arm/arm.mk

```diff
@@ -2,6 +2,7 @@ CFG_LTC_OPTEE_THREAD ?= y
 # Size of emulated TrustZone protected SRAM, 300 kB.
 # Only applicable when paging is enabled.
 CFG_CORE_TZSRAM_EMUL_SIZE ?= 307200
+CFG_ADDR_SPACE_SIZE ?= (1ull << 32)
```

Comment:
Not a big issue, but I'm a bit concerned that you did not add `LPAE` in the variable name, since it applies only when `CFG_WITH_LPAE == y`. @jenswi-linaro what do you think?